applications/luci-app-isc-dhcp/root/usr/share/rpcd/acl.d/luci-app-isc-dhcp.json

```diff
@@ -0,0 +1,8 @@
+{
+	"luci-app-isc-dhcp" : {
+		"description" : "Grant access to LuCI app ISC DHCP",
+		"read" : {
+			"uci": [ "dhcp" ]
+		}
```

Comment:
You will need to add an `"write": { "uci": [ "dhcp" ] }` scope too. While `luci-base` currently implicitly allows uci writes everywhere for backwards compat reasons, it is better to be explicit about the required permissions here.

Reply:
updated